src/parser/template/output.ts

```diff
@@ -14,8 +16,46 @@ export function parseOutput(x: schema.Output): TemplateOutput {
     value: parseExpression(x.Value),
     conditionName: ifField(x, 'Condition', assertString),
     description: ifField(x, 'Description', assertString),
-    exportName: ifField(x, 'Export', (e) =>
-      parseExpression(assertField(e, 'Name'))
-    ),
+    exportName: ifField(x, 'Export', (e) => ifField(e, 'Name', assertString)),
   };
 }
+
+export interface DeCdkCfnOutputProps {
+  value: any;
+  description?: string;
+  exportName?: string;
+  condition?: string;
+}
+
+export class DeCDKCfnOutput extends CfnElement {
+  private readonly value: any;
+  private readonly description?: string;
+  private readonly exportName?: string;
+  private readonly condition?: string;
+  constructor(scope: Construct, id: string, props: DeCdkCfnOutputProps) {
+    super(scope, id);
+    if (props.value === undefined) {
+      throw new Error(
+        `Missing value for CloudFormation output at path "${this.node.path}"`
+      );
+    }
+    this.value = props.value;
+    this.condition = props.condition;
+    this.description = props.description;
+    this.exportName = props.exportName;
+  }
+
+  public _toCloudFormation(): object {
+    return {
+      Outputs: {
+        [this.logicalId]: {
+          Description: this.description,
+          Value: this.value,
+          Export:
+            this.exportName != null ? { Name: this.exportName } : undefined,
+          Condition: this.condition ?? undefined,
+        },
+      },
+    };
+  }
+}
```

Comment:
Not super imporant, but I think this would be better in the `evaluate` module, not the parser.